coderd/database/pubsub/pubsub.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/google/uuid"
 	"github.com/lib/pq"
 	"github.com/prometheus/client_golang/prometheus"
+	"go.uber.org/atomic"
```

Comment:
we are using this elsewhere, but I'd sorta prefer `sync/atomic` in new code so that we have less work cut out for us if we want/need to drop this dependency.